test/expressions/generateExpressionTests.py

```diff
@@ -176,12 +176,15 @@ def parse_signature(signature):
     "lkj_corr_lpdf": [1, None],
     "lkj_corr_log": [1, None],
     "log_diff_exp": [3, None],
-    "log_inv_logit_diff": [1.2, 0.4],
+    "log_inv_logit_diff": [1.2, 0.4],  
     "multinomial_log" : [None, 1],
     "multinomial_lpmf" : [None, 1],
     "multinomial_rng" : [1, None, None],
-    "unit_vector_free" : [1.0],
-
+    "unit_vector_free" : [1.0],    "student_t_cdf" : [0.8, None, 0.4, None],
+    "student_t_cdf_log" : [0.8, None, 0.4, None],
```

Comment:
For student_t arguments 2 and 4 need to be positive, but the others are unconstrained (https://mc-stan.org/docs/2_24/functions-reference/student-t-distribution.html). Why the constraint here?

Reply:
By default all arguments have same values. With y==mu I was getting NaNs.

Reply:
Got it. This is fine then.